core-domain/src/main/java/org/uniprot/core/citation/builder/AbstractCitationBuilder.java

```diff
@@ -57,12 +57,12 @@
         return getThis();
     }
 
-    public @Nonnull B citationXrefsSet(List<DBCrossReference<CitationXrefType>> citationXrefs) {
+    public @Nonnull B citationXrefsSet(List<CrossReference<CitationDatabase>> citationXrefs) {
```

Comment:
Sorry, maybe this is too far...  but, do you think we should rename the property name as well to reflect the change? this will change the property name in the json response and it might be helpful as well... what do you think? I am not sure if this is a good idea or not... Maybe we could discuss with @eddturner  as well... 
I am commenting in the builder, but it will also reflect a change to the interface and impl classes....